pkg/apis/resource/extension_view.go

```diff
@@ -36,14 +37,19 @@ type RouteUpgradeRequest struct {
 
 	model.ResourceUpdateInput `path:",inline" json:",inline"`
 
-	Draft bool `json:"draft,default=false"`
+	Draft           bool `json:"draft,default=false"`
+	ReuseAttributes bool `json:"reuseAttributes,default=false"`
 }
 
 func (r *RouteUpgradeRequest) Validate() error {
 	if err := r.ResourceUpdateInput.Validate(); err != nil {
 		return err
 	}
 
+	if r.ReuseAttributes {
```

Comment:
prefer still validate while reusing attributes, since template and definition may be changed